environment/unit_test.go

```diff
@@ -0,0 +1,210 @@
+package environment
```

Comment:
nit: Go convention, unit tests should be `environment_test.go` if we're testing the environment file, not `unit_test.go`